android/assets/jsons/translations/template.properties

```diff
@@ -550,6 +550,7 @@ Community =
 Close = 
 Do you want to exit the game? = 
 Start bias: = 
+Cancel improvement order = 
```

Comment:
Since you have already added it to the `TileImprovements.json` there is no need to include it into `template.properties`. It will be in translations files generated automatically.

Reply:
Oversight, belonged to the first kludge w/ json entry. Thanks.